shared/chat/conversation/container.js

```diff
@@ -62,21 +53,16 @@ const mapStateToProps = (state: TypedState, {routePath, routeState}) => {
   }
 }
 
-const mapDispatchToProps = (dispatch: Dispatch, {setRouteState, navigateUp}) => ({
+const mapDispatchToProps = (dispatch: Dispatch, {setRouteState, navigateUp}): DispatchProps => ({
   _onAttach: (selectedConversation, inputs: Array<Constants.AttachmentInput>) => { dispatch(navigateAppend([{props: {conversationIDKey: selectedConversation, inputs}, selected: 'attachmentInput'}])) },
   onBack: () => dispatch(navigateUp()),
-  onOpenConversation: (conversationIDKey: Constants.ConversationIDKey) => dispatch(Creators.openConversation(conversationIDKey)),
-  startConversation: (users: Array<string>) => dispatch(Creators.startConversation(users, true)),
 })
 
 const mergeProps = (stateProps: StateProps, dispatchProps: DispatchProps) => {
   return {
     ...stateProps,
```

Comment:
i've been moving away from doing ...stateProps (see below). i think its likely better to be very explicit about what gets passed down so we don't get any props leaking down (like the _ ones)